android/app/src/main/java/com/chorestory/fragment/ChildClanFragment.java

```diff
@@ -0,0 +1,22 @@
+package com.chorestory.fragment;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.chorestory.R;
+
+public class ChildClanFragment extends Fragment {
+
+    @Nullable
```

Comment:
I think this never returns `null`, https://developer.android.com/reference/android/view/LayoutInflater.html#inflate(org.xmlpull.v1.XmlPullParser,%2520android.view.ViewGroup,%2520boolean)